packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -59,6 +59,9 @@ export type BrandedKeyContent<TKey extends BrandedKey<unknown, any>> = TKey exte
 	: never;
 
 /**
+ * Stores arbitrary, user-defined data on an {@link Anchor}.
+ * This data is preserved over the course of that anchor's lifetime.
```

Comment:
not sure this sentence has much value, i'm ok removing it.

@taylorsw04 FYI. On second glance, the docs on the `anchorSlot` function (which the `slots` property points to) are pretty reasonable. I probably just went to this type first, saw nothing, and moved on. Makes more sense to me to document the type with its usage for things like this, anyway.